math/least_common_multiple.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2020 @author tjgurwara99
+ * @file
+ *
+ * A basic implementation of LCM function
+ */
+
+#include <cassert>
+#include <iostream>
+
+/**
+ * Function for finding greatest common divisor of two numbers.
+ * @params two integers x and y whose gcd we want to find.
+ * @return greatest common divisor of x and y.
+ */
+unsigned int gcd(unsigned int x, unsigned int y) {
+    if (x == 0) {
+        return y;
+    }
+    if (y == 0) {
+        return x;
+    }
+    if (x == y) {
+        return x;
+    }
+    if (x > y) {
+        // The following is valid because we have checked whether y == 0
+
+        int temp = x / y;
+        return gcd(y, x - temp * y);
+    }
+    // Again the following is valid because we have checked whether x == 0
+
+    int temp = y / x;
+    return gcd(x, y - temp * x);
+}
+
+/**
+ * Function for finding the least common multiple of two numbers.
+ * @params integer x and y whose lcm we want to find.
+ * @return lcm of x and y using the relation x * y = gcd(x, y) * lcm(x, y)
```

Comment:
You can use LaTeX syntax for math formulae in the documentation:
```Latex
\f$x \cdot y = \mathrm{gcd}(x, y) \cdot /mathrm{lcm}(x, y)\f$
```
[Example output](https://kvedala.github.io/C-Plus-Plus/d7/d6a/bisection__method_8cpp.html#details)

Reply:
I didn't know about this feature of Doxygen, thanks for pointing it out!